hydromt/gis_utils.py

```diff
@@ -547,3 +550,15 @@ def write_map(
         if crs is not None:
             with rasterio.open(raster_path, "r+") as dst:
                 dst.crs = crs
+
+
+def create_vrt(path: str, fname: str):
```

Comment:
This function needs a docstring and needs to be added to the api.rst.
You also assume there is a filelist.txt in path. I think that's a fine default, but it would be nice to add the option to generate this file using glob. Either way this should be made clear in the docstring.

Reply:
Done